app/client/assets/schools.csv

```diff
@@ -74,86 +95,126 @@ Brampton Centennial Secondary School
 Brandeis University
 Briar Cliff University
 Brigham Young University
+Brno University of Technology
 Brock University
+Bronx Community College, CUNY
 Brookdale Community College
-Brooklyn Technical High School
 Brooklyn College, CUNY
-Medgar Evers College, CUNY
+Brooklyn Technical High School
 Brookwood High School
 Brown University
+Bryan University
+Bryn Mawr College
 Bucknell University
 Business Academy Aarhus
-COMSATS Institute of Information Technology
+Butte College
+Cabrillo College
 Caldwell University
+California Baptist University
+California Christian College
+California College of the Arts
+California College San Diego
 California High School
 California Institute of Technology
+California Institute of the Arts
+California Lutheran University
+California Maritime Academy
+California Polytechnic State University San Luis Obispo
 California Polytechnic State University, San Luis Obispo
+California State Polytechnic University Pomona
 California State Polytechnic University, Pomona
+California State University Bakersfield
+California State University Channel Islands
+California State University Chico
+California State University Dominguez Hills
+California State University East Bay
+California State University Fresno
+California State University Fullerton
+California State University Long Beach
+California State University Los Angeles
+California State University Monterey Bay
+California State University Northridge
+California State University Sacramento
+California State University San Bernardino
+California State University San Marcos
+California State University Stanislaus
+California State University, Bakersfield

```

Comment:
there seems to be quite a lot of repeated entries for CSUs, is this intentional?